### PR TITLE
fix zones doc formats

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Describe your changes. This will most likely be a copy-paste of your commit
  message(s), which should be descriptive and informative.

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [x] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-125562](https://oktainc.atlassian.net/browse/OKTA-125562)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:

- *Tag reviewer(s)*
@jakubvul-okta

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@rchild-okta my change to the TOC is trivial, and I tested locally, but just in case, if you want to review, go right ahead.
@jakubvul-okta I just added some headings. This topic isn't quite the same as others, but closer now.

File: _source/_docs/api/resources/zones.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: docs_page
-title: Zones API
+title: Zones
 ---
 
 ## Zones API
```
Comment (johngronberg-okta) [inline _source/_docs/api/resources/zones.md]: Should this have one # instead of two?

Comment (mystiberry-okta) [inline _source/_docs/api/resources/zones.md]: In the current CSS, pound and pound pound are indistinguishable. I leave them as pound pound  so people aren't fooled into using #.

Comment (johngronberg-okta) [inline _source/_docs/api/resources/zones.md]: All of the other API reference pages use a single # for the first heading. So, if that CSS ever changes, this page will look different than all of the others. I would suggest either matching this file to the convention used elsewhere, or update the other pages for consistency.

Comment (mystiberry-okta) [inline _source/_docs/api/resources/zones.md]: Will fix this page (I thought I had some others with ##)

**Comment by mystiberry-okta** [comment]
@rchild-okta Can you please look at this? I did as you asked and changed the base from master to weekly and now there's a bunch of unrelated commits in this PR. Guessing I was suppposed to wait for you to rebase master first?

**Comment by rchild-okta** [comment]
The problem is that the original PR is not rebased off of weekly - in general, the workflow should be to check out a branch off of weekly and then push to weekly (it's the same workflow as master, except that you're using weekly as your master).

Do this:
```bash
[format-doc-zones]$ git fetch origin
[format-doc-zones]$ git rebase -i origin/weekly
[format-doc-zones]$ git push -f origin format-doc-zones:format-doc-zones
```

**Comment by rchild-okta** [comment]
And it looks like you have some extra stuff in there, so when you do the rebase just delete the lines with commits you don't recognize.

**Comment by mystiberry-okta** [comment]
@rchild-okta I (tried to!) follow your instructions but the unwanted commits are still there. I just seem to be adding new commits....I did delete all the non-zones commits from the rebase but am still doing something wrong. Not sure what mistake I made.

**Comment by mystiberry-okta** [comment]
whew! I'm learning so much! @rchild-okta if you could review, or @johngronberg-okta . Since I did so much merging and rebasing, I rebuilt local and retested.

**Pull request merged**